src/index.js

```diff
@@ -301,7 +301,7 @@ module.exports.warmUp = (event, context, callback) => {
       handler: this.pathHandler,
       memorySize: this.warmup.memorySize,
       name: this.warmup.name,
-      runtime: 'nodejs6.10',
+      runtime: 'nodejs8.10',
```

Comment:
Have you tested this in a real AWS deployment?

In theory, updating to 8.10 is harmless but that's not true.
I recently had many problems with Promises and Node 8.10 in Lambda in several projects.
Essentially async-await worked but not plain promises.

Reply:
i have not updated runtime for this function to 8.10. i wanted to get this change in and then test. i dont see why there would be any issue. i have multiple lambda fns in production on 8.10 and had no problems. (my fns always are transpiled anyway)

there is performance improvements over 6.10 and 6.x line has moved into maintenance mode

Reply:
There shouldn't be any problem updating to 8.10.
The keyword here is "shouldn't".

In my experience, there are.
I don't think that lambdas use a standard node.js distribution so you can't rely on things working exactly as they would on your machine.

One of the issues that I've noticed is the 8.10 failing with promises instead of asyn-await even though it worked locally using serverless-offline plugin and such.

Other examples are circular dependencies. They work on node but will break in Lambdas.

A lot of people is not transpiling the code before pushing to lambdas. So I wouldn't be totally comfortable merging this until it's tested that this works on a real AWS deployment.

Reply:
Why don't we just open it up to a user-configured value, perhaps even defaulted to `6.10`, then let us validate it in production (which I'll do immediately) and then we can later bump the default?

Reply:
And side note ... I've run lots of lambdas in both 6.10 and 8.10. 8.10 works just fine for everything we've thrown at it so far!!!

Reply:
We could do a user config option and default to `6.10`. @juanjoDiaz what do you think?